app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java

```diff
@@ -44,6 +46,7 @@
     private ContainerMessagesViewModel mContainerMessagesViewModel;
     private MessageMapEntitiesViewModel mMessageMapEntitiesViewModel;
     private UsersLocationViewModel mUserLocationViewModel;
+    private Handler mBackgroundHandler;
```

Comment:
Obviously, if the name is still bad in your opinion we can try and think of something else